client/controllers/graph.controller.js

```diff
@@ -1,16 +1,85 @@
 (function() {
-    
     "use strict";
     
     angular
-        .module("graph", [])
+        .module("smitty.graph", [])
         .controller("GraphController", GraphController);
     
-    function GraphController($scope, $http){ 
+    GraphController.$inject = ["canvasService"];
+    
+    function GraphController(canvasService) {
+        var vm = this; // View Model
+        
+        var nextVertexId = 0;
+        var vertexColor = "#CC8193";
```

Comment:
I made vertex color a variable so that you can easily change it in the future if you want. This is more maintainable than having a string literal getting passed to `canvasService`.